pkg/virt-handler/virtwrap/cache/cache.go

```diff
@@ -70,7 +70,7 @@ var CrashedReasonTranslationMap = map[libvirt.DomainCrashedReason]api.StateChang
 }
 
 // NewListWatchFromClient creates a new ListWatch from the specified client, resource, namespace and field selector.
-func newListWatchFromClient(c virtwrap.Connection, events ...int) *cache.ListWatch {
+func NewListWatchFromClient(c virtwrap.Connection, events ...int) *cache.ListWatch {
```

Comment:
Not that we were taking care about exports so far, but are you deliberately exporting that function?